modules/create_dashboard_html.nf

```diff
@@ -0,0 +1,31 @@
+// Create data guide
+process create_dashboard_html {
```

Comment:
Are we planning on removing this from our [consortium release/public release step here](https://github.com/Sage-Bionetworks/Genie/blob/9295c5c2aec47a8a62282e26eac56736a4925edb/bin/database_to_staging.py#L294)?

Reply:
It could be a good first step in simplifying the codebase!

It would be good to be able to pull the dependencies all out, but.. this is probably a good iterative addition.